cips/cip-rename_da_to_dp.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Rename data availability to data publication
+description: Renaming data avaiability to data publication to better reflect the message
+author: msfew (@fewwwww) <msfew@hyperoracle.io>, Kartin <kartin@hyperoracle.io>, Xiaohang Yu (@xhyumiracle)
+discussions-to: https://forum.celestia.org/t/informational-cip-rename-data-availability-to-data-publication/1287
+status: Draft
+type: Informational
+created: 2023-11-06
+---
+
+## Abstract
+
+The term `data availability` isn't as straightforward as it should be and could lead to misunderstandings within the community. To address this, this CIP proposes replacing `data availability` with `data publication`.
+
+## Motivation
+
+The term `data availability` has caused confusion within the community due to its lack of intuitive clarity. For instance, in Celestia's Glossary, there isn't a clear definition of `data availability`; instead, it states that `data availability` addresses the question of whether this data has been published. Additionally, numerous community members have misinterpreted `data availability` as meaning `data storage`.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The term `data availability` is RECOMMENDED to be renamed to `data publication`.
+
+## Rationale
+
+Motivations:
+- `Data publication` aligns more precisely with the intended meaning, which revolves around whether data has been published.
+- The community already favors and commonly uses the term `data publication`.
```

Comment:
Any links or proof to help support your case?

Reply:
Yes, should I add them to the CIP itself, or maybe add them to the discussion post?

Here's a few links to help support the case:
- [Poll by Modular Media prefers "data publication" to "data availability"](https://twitter.com/modularmedia_/status/1700182744884510760)
- [AltLayer's founder used data publication to refer to DA](https://twitter.com/jiayaoqi/status/1681044622099861504)
- [Namada uses both terms interchangeably](https://namada.net/blog/namada-and-celestia-exploring-a-path-toward-shielded-data-availability)
- [Mustafa thinks "we should commit to officially renaming data availability to data publication"](https://twitter.com/musalbas/status/1696521530409394253)

Reply:
Yes add them to the discussion link in the forum